### ISSUE TITLE
how Can it show emulator in web?

### ISSUE THREAD

**Issue by HCDarren** [google/android-emulator-container-scripts#135]
[./create_web_container.sh] and [docker-compose -f js/docker/docker-compose.yaml up] is run successful. Point mine browser to localhost. my get a warning due to the usage of the self signed certificate. I've authorized it, but it is not show anything.

**Comment by HCDarren** [comment]
![](https://user-images.githubusercontent.com/8591425/79977955-b5b83100-84d1-11ea-9b8d-eaabcecc2d3b.png)

**Comment by pokowaka** [comment]
Hi,

You should see at least a login screen. If possible could you share the logs?

**Comment by HCDarren** [comment]
this is my logs, thank you!

[sudo ./create_web_container.sh]
[create_web_container.log](https://github.com/google/android-emulator-container-scripts/files/4532882/create_web_container.log)

[docker-compose -f js/docker/docker-compose.yaml up]
[launch.log](https://github.com/google/android-emulator-container-scripts/files/4532886/launch.log)

**Comment by pokowaka** [comment]
I'm not sure what is happening.  From the logs I can gather that the emulator rebooted once:

```
emulator_1     | kernel: [    5.899284] Restarting system with command ''.
emulator_1     | kernel: [    5.900724] reboot: machine restart
emulator_1     | kernel: [    0.000000] Initializing cgroup subsys cpu
```
However logcat got launched, so eventually the system recovered.

I noticed that the video bridge was not launched, so that is unexpected and might be a reason why you are not seeing any video. 

I saw from the logs that you are using an 'M' image. I will see if I can reproduce your issue with that version.
